interfaces/builtin/unity8_calendar.go

```diff
@@ -0,0 +1,163 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const unity8CalendarPermanentSlotAppArmor = `
+# Description: Allow operating as the EDS service. Reserved because this
+#  gives privileged access to the system.
+# Usage: reserved
+
+# DBus accesses
+#include <abstractions/dbus-session-strict>
+dbus (bind)
+	bus=session
+	name="org.gnome.evolution.dataserver.Calendar7",
+dbus (bind)
+	bus=session
+	name="org.gnome.evolution.dataserver.Subprocess.Backend.Calendar*",
+dbus (bind)
+	bus=session
+	name="com.canonical.SyncMonitor",
+
+# Allow traffic to/from our path and interface with any method for unconfined
+# clients to talk to our calendar services.
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/CalendarFactory
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/CalendarView/**
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/Subprocess/**
+	interface=org.gnome.evolution.dataserver.Calendar
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/Subprocess/Backend/Calendar/**
+	peer=(label=unconfined),
```

Comment:
Since slot implementations require a snap declaration to be in the store, I'm 'ok' with this access to unconfined. Note that my comment was not about unconfined talking to the slot (which should always be fine), it is about the slot perhaps being able to abuse its privilege and send weird stuff to unconfined. Can you try out these rules with just 'receive' and report back? They might be required, but it is worth check if they're not.